### PR TITLE
small fix in utils_misc.py

### PR DESCRIPTION
**Pull request by PyLearner**
small fix in utils_misc.py

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -500,7 +500,7 @@ def wait_for(func, timeout, first=0.0, step=1.0, text=None):
     :param text: Text to print while waiting, for debug purposes
     """
     start_time = time.time()
-    end_time = time.time() + timeout
+    end_time = time.time() + int(timeout)
```
Comment (PandaWei) [inline virttest/utils_misc.py]: I really don't see any reason for convert it to int.
Although the timeout is set to int, such as 60, 120, in most cases.
it doesn't mean that we can't set float value to timeout.
the default of step is 1.0. 
so i think float value is supported for timeout

Comment (lmr) [inline virttest/utils_misc.py]: @PandaWei, correct

Comment (lmr) [inline virttest/utils_misc.py]: Maybe the author @PyLearner meant to convert to `float` instead of `int`?

**Comment by lmr** [comment]
@PyLearner, you also need to rebase your branch to clean it up and leave only the commits you actually want to contribute with.

**Comment by PyLearner** [comment]
@lmr,@PandaWei  ,sorry to reply late,i made the wrong change ,i will close it ,thanks for your comment!

File: shared/cfg/guest-os/Linux/Ubuntu/12.04-server.i386.cfg [hunk 2]
```diff
@@ -1,4 +1,5 @@
 - 12.04-server.i386:
+    grub_file = /boot/grub/grub.cfg
```
Comment (ypu) [inline shared/cfg/guest-os/Linux/Ubuntu/12.04-server.i386.cfg]: Please config this in Ubuntu.cfg. As the are all the same so don't need update in separated cfg files.

**Comment by ypu** [comment]
@PyLearner you don't need close it. Just update your pullreq branch will be OK.

**Pull request closed without merging**